cobalt/browser/application.cc

```diff
@@ -1207,6 +1229,7 @@ void Application::OnApplicationEvent(SbEventType event_type,
     case kSbEventTypeStop:
       LOG(INFO) << "Got quit event.";
       if (watchdog) watchdog->UpdateState(base::kApplicationStateStopped);
+      AddCrashHandlerApplicationState(base::kApplicationStateStopped);
```

Comment:
Do we want to track Conceal/Reveal Blur/Focus state as well ? Perhaps we should just store whichever the last lifecycle transition event was ?

Reply:
It does track all of the different application states, it just does it in line 1351 below. The `kApplicationStateStopped` is an one off because of the shortcircuit `quit()` that exits this logic.